### ISSUE TITLE
State value to set  has to be type "string" but received type "number"

### ISSUE THREAD

**Issue by Feuer-sturm** [StrathCole/ioBroker.rainbird#4]
Admin 5.0.13
js-controller 3.3.6

Mit dem Einsatz vom js-controller 3.3.x kommen beim Start von Rainbird folgende Meldungen:

```
03 May 2021 00:11:03.160
 rainbird.0 (23623) State value to set for "rainbird.0.device.major" has to be type "string" but received type "number"
03 May 2021 00:11:03.156
 rainbird.0 (23623) State value to set for "rainbird.0.device.minor" has to be type "string" but received type "number"
03 May 2021 00:11:03.154
 rainbird.0 (23623) State value to set for "rainbird.0.device.model" has to be type "string" but received type "number"
```

**Comment by martin-ef** [comment]
although closed. I still get the exact same error using v0.2.2 - How can I fix it?

**Comment by Feuer-sturm** [comment]
@martin-ef The fixed version 0.2.3 is only available on github and not on npm yet. You can only wait if 0.2.3 is available on npm (#14) or if you know what you are doing update your adapter from github

**Comment by martin-ef** [comment]
Ah okay. Will try to figure out how to install the github-version. 

Thank you very much for the fast reply!
